File: _drlecode_simulation_framework/entities/Entity.java
package entities;

public class Entity
{
	public double ID;
	public boolean warp = false;
	public boolean modify = false;
	public boolean sense = false;
	//This distance is used to sort entity base on this entity distance to a target agent.
	public double distance = 0;
	public boolean remove = false;
	
	public Entity (double t)
	{
		this.ID = t;
	}
	
}

File: agent-simulation-react/public/simulation-legacy/entity.js
export class Entity { // where it all started
    constructor(ID) {
        this.ID = ID;
        this.warp = false;
        this.modify = false;
        this.sense = false;
        // distance used to sort entity base;
        this.distance = 0;
        this.remove = false;
    }
}